packaging/os/pkgutil.py

```diff
@@ -63,10 +63,10 @@
 import pipes
 
 def package_installed(module, name):
-    cmd = [module.get_bin_path('pkginfo', True)]
+    cmd = ['pkginfo']
```

Comment:
I definitely see your point. I was trying to make the module's current code more consistent -- notice that it only uses module.get_bin_path in the package_installed() function and in all the other 'package_' functions does what you're criticizing.

Your comments suggest that it may be better to remove the run_command function entirely, since all it effectively does is do a module.get_bin_path for calling functions. Do you think this is a better way to go? The only downside I see to this is adding several redundant lines of code where I do this in each function that runs a command:

cmd = [ module.get_bin_path(COMMAND, True, PATH) ]
cmd.extend(ADDITIONAL COMMAND ARGS)

Is it not better to do it in just one place?

-- Thanks